inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+archives“) and browse for the example project you have downloaded
+before. Finally, click the grey **upload-**Butten right next to the blue
+folder to upload the selected project.
 
-Next, switch to the *Documents* tab and press *choose files* to select a plain text file from
-your local harddisk (the file should be in UTF-8 encoding). As part of the import process, 
-{product-name} automatically processes the file to identify sentence and token boundaries.
+* The project has now been added. Choose it by clicking on it and you
+will be on the dashboard of this project. Select *Annotation* on the top
+left and choose _pets1.tsv_ (the first file) in the window which now
+opened.
+* You will see several annotations already. If you import a documents
+without annotations yet, there will be none. But in the example
+projects, we have added some annotations already to show you some
+examples of annotations.
+* To create your first annotation, let‘s start with Part of Speech
+Tagging: Select *POS* from the *Layer* dropdown on the top right. When
+you click on a word in the text (e.g. „does“, in line 4), an annotation
+is immediately created and you can edit its details in the panels at the
+right (e.g. choose „VERB“ in the PosValue-dropdown to lable it as a
+verb).
 
-Use the *Home* link at the top of the screen to return to the main menu and select *Annotation*
-to open your text file in the annotation editor.
+Congratulations! You have created your first annotation!
 
-To create your first annotation, select *Named entity* from the *Layer* dropdown menu on the right
-side of the screen. Then, use the mouse to select a word in the *Annotation* area. When you release
-the mouse button, the annotation is immediately created and you can edit its details in the right
-sidebar.
+* Let‘s examine the right panel:
 
-Congratulations! You have created your first annotation project.
+* {blank}
+** You find the field „Layer“ on top and „Annotation“ below. In the
+*Layer*-dropdown, you can choose the layer you want to annotate with as
+we yust did. You find several default layers linke Part Of Speech or
+Named Entity already. You can also adjust them and create own layers in
+the project settings (we will deal with the project settings later on).
+INCEpTION supports span layers (annotating a span in the text, e.g. from
+the 51th character („letter“) in the text to the 55th) and relation
+layers (annotating two span annotations). For span layers, the default
+is to always annotate one or more tokens („words“) but you can adjust to
+character level or sentence level. For more details, see the main
+documentation. LINK EINFÜGEN.
+** In the field *Annotation* below, you see the details of a selected
+annotation. It shows the layer the annotation is made on (field „layer“)
+and what has been annotated (field „text“). Below, you can see and
+modify what has been entered for each of the so called *Features*. Each
```

Comment:
That should come earlier, before annotating

Reply:
I think it's fine. Right away, we examine the panel to understand what we did. First doing something and then explaining it right away is one valid option to teach, besides first explaining and then doing it (where the explanation is harder to understand). I guess it's a matter of taste. But I changed the sentence before to lead our users well: "Now, let‘s examine the right panel to understand what we have done:"

Reply:
i gave an additional note before that we will see after what to do with the features ("details").